Mathlib/Analysis/Convex/Extreme.lean

```diff
@@ -113,6 +116,19 @@ theorem mem_extremePoints : x ∈ A.extremePoints 𝕜 ↔
     x ∈ A ∧ ∀ᵉ (x₁ ∈ A) (x₂ ∈ A), x ∈ openSegment 𝕜 x₁ x₂ → x₁ = x ∧ x₂ = x :=
   Iff.rfl
 
+/-- In order to prove that a point `x` is an extreme point of a set `A`,
+it suffices to show that `x ∈ A`
+and for any `x₁`, `x₂` such that `x` belongs to the open segment `(x₁, x₂)`, we have `x₁ = x`.
+
+The definition of `extremePoints` also requires `x₂ = x`, but this condition is redundant. -/
+theorem mem_extremePoints_iff_left : x ∈ A.extremePoints 𝕜 ↔
+    x ∈ A ∧ ∀ x₁ ∈ A, ∀ x₂ ∈ A, x ∈ openSegment 𝕜 x₁ x₂ → x₁ = x := by
```

Comment:
Please start multi-line return type on a new line
```suggestion
theorem mem_extremePoints_iff_left :
    x ∈ A.extremePoints 𝕜 ↔
      x ∈ A ∧ ∀ ⦃x₁⦄, x₁ ∈ A → ∀ ⦃x₂⦄, x₂ ∈ A → x ∈ openSegment 𝕜 x₁ x₂ → x₁ = x := by
```
and also keep the binders the same as in the definition, for practicality.

Reply:
I copied this from the previous lemma.